firewood/src/merkle/node/branch.rs

```diff
@@ -179,3 +212,59 @@ impl BranchNode {
             .unwrap()
     }
 }
+
```

Comment:
I will have to review this later to verify it didn't change.

Reply:
You actually do not have to verify that it didn't change. What you have to verify is that we have tests that make sure that things that we serialize can also be deserialized since we have no burden of backward compatibility (yet).

Reply:
🤔 , now that I say that... I should have explicit tests for this. This is actually tested in all the `merkle.rs` tests because of [this line](https://github.com/ava-labs/firewood/blob/313cae7852d5ea66af1b43452634a4639bd1dfec/firewood/src/merkle.rs#L1682), but that's pretty fragile.

I created a new issue: https://github.com/ava-labs/firewood/issues/375